packages/collector/test/tracing/protocols/http/proxy/test.js

```diff
@@ -187,15 +186,14 @@ mochaSuiteFn('http with proxy', function () {
           retry(() =>
             agentControls.getSpans().then(spans => {
               expectAtLeastOneMatching(spans, [
-                span => expect(span.n).to.equal('node.http.client'),
+                span => expect(span.n).to.equal('node.http.server'),
                 span => expect(span.error).to.not.exist,
                 span => expect(span.ec).to.equal(1),
                 span => expect(span.f.e).to.equal(String(expressProxyControls.getPid())),
                 span => expect(span.f.h).to.equal('agent-stub-uuid'),
                 span => expect(span.async).to.not.exist,
-                span => expect(span.data.http.error).to.be.a('string'),
                 span => expect(span.data.http.method).to.equal('POST'),
-                span => expect(span.data.http.url).to.equal('http://10.123.456.555:49162/foobar')
+                span => expect(span.data.http.url).to.equal('/callNonExistingTarget')
```

Comment:
q: Why?

Reply:
Ignore this comment. See other [conversation](https://github.com/instana/nodejs/pull/1004#discussion_r1455367171).